### PR TITLE
Code for AI on GKE guide series

### PR DESCRIPTION
**Pull request by ganochenkodg**
- Kustomize patches to run various quantized models in vLLM and TGI runtimes.

### CODE REVIEW

File: ai-ml/llm-serving-gemma/vllm/vllm-2b-awq/patch.yaml [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+    spec:
+      containers:
+      - name: inference-server
+        args:
+        - --model=$(MODEL_ID)
+        - --tensor-parallel-size=1
+        - --quantization=awq
+        env:
+        - name: MODEL_ID
+          value: dganochenko/gemma-2b-AWQ
```
Comment (brandonroyal) [inline ai-ml/llm-serving-gemma/vllm/vllm-2b-awq/patch.yaml:33]: Please change this to `value: google/gemma-2b-AWQ` so it points to the right repository.

File: ai-ml/llm-serving-gemma/vllm/vllm-2b.yaml
Comment (brandonroyal) [inline ai-ml/llm-serving-gemma/vllm/vllm-2b.yaml:1]: Small detail but can we remove this file change?

File: ai-ml/llm-serving-gemma/vllm/vllm-7b-awq/patch.yaml [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+    spec:
+      containers:
+      - name: inference-server
+        args:
+        - --model=$(MODEL_ID)
+        - --tensor-parallel-size=1
+        - --quantization=awq
+        env:
+        - name: MODEL_ID
+          value: dganochenko/gemma-7b-AWQ
```
Comment (brandonroyal) [inline ai-ml/llm-serving-gemma/vllm/vllm-7b-awq/patch.yaml:33]: Please change to `value: google/gemma-7b-AWQ` to ensure we're pointing at the right repository

**Review by brandonroyal — CHANGES_REQUESTED** [review]

**Review by brandonroyal — APPROVED** [review]

File: ai-ml/llm-serving-gemma/vllm/vllm-7b-awq/patch.yaml (hunk 2 above)
Comment (ganochenkodg) [reply ai-ml/llm-serving-gemma/vllm/vllm-7b-awq/patch.yaml]: it's done already

File: ai-ml/llm-serving-gemma/vllm/vllm-2b-awq/patch.yaml (hunk 1 above)
Comment (ganochenkodg) [reply ai-ml/llm-serving-gemma/vllm/vllm-2b-awq/patch.yaml]: it's done already

File: ai-ml/llm-serving-gemma/vllm/vllm-2b.yaml
Comment (ganochenkodg) [reply ai-ml/llm-serving-gemma/vllm/vllm-2b.yaml:1]: Sure, done

**Comment by brandonroyal** [comment]
Looks good.  @TarasRudko @ganochenkodg Can we mark this as ready for review?

**Review by brandonroyal — APPROVED** [review]

**Review by bourgeoisor — APPROVED** [review]
Looks good from my end

**Pull request merged**